src/test/regress/expected/foreign_key_to_reference_table.out

```diff
@@ -0,0 +1,785 @@
+SET citus.shard_replication_factor TO 1;
+SET citus.shard_count TO 8;
+CREATE SCHEMA fkey_reference_table;
```

Comment:
Can you move create schema & set search path to the first line? (that's probably more readable)

Also, in general, we start with a file comment in the regression test as well.